webapp/components/interop.js

```diff
@@ -1,1408 +1,7 @@
-/**
- * Copyright 2022 The WPT Dashboard Project. All rights reserved.
- * Use of this source code is governed by a BSD-style license that can be
- * found in the LICENSE file.
- */
+import { InteropDashboard } from "./interop-dashboard.js";
```

Comment:
The copyright comment needs to be placed back at the top of this file.

Reply:
Fixed!